### ISSUE TITLE
not every enrolled course shows up

### ISSUE THREAD

**Issue by apalm1341** [FaisalUmair/udemy-downloader-gui#76]
not every enrolled course shows up in app/program. Are all enrolled courses supposed to show up in app?

**Comment by FaisalUmair** [comment]
Yes all enrolled courses are supposed to show up in the app. The results returned by Udemy are paginated, so you need to click on 'Load more' button in order to load results from more pages.

If you still don't see any course, please let me know.

**Comment by karanmht9** [comment]
Hi Team

I am not able to see enrolled courses in udeler app when i logged in mine account previously it was showing.
